client/src/components/Navbar.js

```diff
@@ -10,13 +10,13 @@ const Navbar = () => {
         <nav className="flex justify-between items-center">
             {/* main nav - centered */}
             <div className='text-center text-xl text-white flex-1 font-medium'>
-                <a className="mr-8" href="">Home</a>
-                <a className="mr-8" href="">About</a>
+                <a className="mr-8" href="/">Home</a>
+                <a className="mr-8" href="/">About</a>
                 <a href="http://localhost:3000/register">Trip Registration</a>
```

Comment:
nit: you don't need to put `href="http://localhost:3000/register"` you can do `href="/register"` instead